doc/issue_impact.md

```diff
@@ -0,0 +1,25 @@
+# Issue Impacts
+
+Axe-core assigns an impact according to our assessment of the likely impact of an issue on a user with a disability that would be affected by this issue.
+
+In any given context the actual impact for the user could be lower and in some instances, it could be higher.
+
+For this reason, we encourage users of tools to evaluate each individual issue and assess the impact in the context of their application or content.
```

Comment:
This seems like a single thought/paragraph to me. 

```suggestion
Axe-core assigns an impact according to our assessment of the likely impact of an issue on a user with a disability that would be affected by this issue. In any given context the actual impact for the user could be lower; in some instances, it could be higher. For this reason, we encourage users of tools to evaluate each individual issue and assess the impact in the context of their application or content.
```